A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -352,12 +352,47 @@ var A320_Neo_UpperECAM;
                 }
             }
         }
+        getLoWingTankActions(_tankSide) {
+            return [
+                {
+                    style: "remark-indent-multiLine1",
+                    message: "IF NO FUEL LEAK",
+                },
+                {
+                    style: "remark-indent-multiLine2",
+                    message: "AND IMBALANCE",
+                },
+                {
+                    style: "action",
+                    message: "FUEL X FEED",
+                    action: "ON",
+                    isCompleted: () => this.getCachedSimVar("A:FUELSYSTEM VALVE SWITCH:3", "Bool"),
+                },
+                {
+                    style: "action",
+                    message: `${_tankSide} TK PUMP 1`,
+                    action: "OFF",
+                    isCompleted: () => {
+                        return (_tankSide == "R") ? !SimVar.GetSimVarValue("FUELSYSTEM PUMP SWITCH:3", "boolean") : !SimVar.GetSimVarValue("FUELSYSTEM PUMP SWITCH:2", "boolean");
```

Comment:
Any reason not to use getCachedSimVar here and all the others below?

Reply:
Thanks! adopted.